### PR TITLE
Write-Only web info

### LINKED ISSUE #22: Write-Only mode not described
https://open-cas.github.io/cache_configuration.html shows different cache modes but does not include newly added Write-Only mode.

### PR DESCRIPTION
**Pull request by Ostrokrzew**
**cache_configuration**: added description of Write-Only mode.
This fixes #22 issue.

Signed-off-by: Slawomir_Jankowski <slawomir.jankowski@intel.com>

### CODE REVIEW

File: pages/basic_concepts/cache_configuration.md [hunk 1]
```diff
@@ -17,20 +17,21 @@ also if data stored on **cache** should always be coherent with data
...
 [**dirty data**](/cache_operations.html)).
 
-Currently there are five **cache modes** supported by OCF:
+Currently there are six **cache modes** supported by OCF:
 - *Write-Through (WT)*,
 - *Write-Back (WB)*,
 - *Write-Around (WA)*,
 - *Write-Invalidate (WI)*,
 - *Pass-Through (PT)*.
+- *Write-Olny (WO)*.
```
Comment (robertbaldyga) [inline pages/basic_concepts/cache_configuration.md]: s/Olny/Only

File: pages/basic_concepts/cache_configuration.md [hunk 2]
```diff
@@ -75,6 +75,14 @@ contained a **dirty data** before switching to *Pass-Through* mode, then
 from **cache storage** until all the cache lines will be
 [**cleaned**](/cache_operations.html).
 
+### Write-Only
+In *Write-Only* mode, the **cache engine** writes the data exactly like
+in *Write-Back* mode and still there is a risk of data loss
```
Comment (robertbaldyga) [inline pages/basic_concepts/cache_configuration.md]: I'd rephrase it to first explain how it works (for both read and write), then describe the benefits (write speed increase), and at the very end inform about the consequences (reads not accelerated, potential data loss).

File: pages/basic_concepts/cache_configuration.md (hunk 1 above)
Comment (robertbaldyga) [inline pages/basic_concepts/cache_configuration.md]: Maybe also place it before PT.

Comment (Ostrokrzew) [reply pages/basic_concepts/cache_configuration.md]: done

Comment (Ostrokrzew) [reply pages/basic_concepts/cache_configuration.md]: done

File: pages/basic_concepts/cache_configuration.md (hunk 2 above)
Comment (Ostrokrzew) [reply pages/basic_concepts/cache_configuration.md]: Rephrased, but waiting for next review.
